generate_localization.sh

```diff
@@ -30,6 +30,6 @@ do
     for f in $(ls $SRC_DIR)
     do
         echo "  generate: $TARGET_DIR/$lang/$f"
-        grep -v -E $filter $SRC_DIR/$f | sed $command > $TARGET_DIR/$lang/$f
+        grep -v -E $filter $SRC_DIR/$f | sed -e "$command" -e 's|import secret from "\.\./wallet.json"|import secret from "\.\./\.\./wallet.json"|g' > $TARGET_DIR/$lang/$f
```

Comment:
nit: `sed -e "$remove_lang_prefix" -e "$replace_wallet_path"` may be reader friendly